src/app/trainer/dashboard/clients/page.jsx

```
@@ -70,65 +67,131 @@ export default function ClientsPage() {
+  }, [sortOption, sortOrder]);
+
+  // Function to sort clients
+  const sortClients = (clientsToSort, option, order) =>
+    [...clientsToSort].sort((a, b) => {
+      let comparison = 0;
+
+      switch (option) {
+        case "name":
+          const aName = `${a.client?.clientProfile?.firstName || ""} ${
+            a.client?.clientProfile?.lastName || ""
+          }`;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)